frontend/src/libs/packages/config/config.package.ts

```diff
@@ -21,6 +21,9 @@ class Config implements IConfig {
       API: {
         ORIGIN_URL: import.meta.env.VITE_APP_API_ORIGIN_URL as string,
       },
+      FACEBOOK: {
+        APP_ID: import.meta.env.VITE_APP_FACEBOOK_APP_ID as string,
+      },
```

Comment:
I guess google auth env variable was missed and is used directly,
Could you please check it and add it if it was missed?

Reply:
I've added it.